x/bank/types/msgs_test.go

```diff
@@ -0,0 +1,283 @@
+package types_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/suite"
+
+	tmproto "github.com/tendermint/tendermint/proto/tendermint/types"
+
+	"github.com/cosmos/cosmos-sdk/baseapp"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/certikfoundation/shentu/v2/simapp"
+	"github.com/certikfoundation/shentu/v2/x/bank/types"
+)
+
+var (
+	acc1 = sdk.AccAddress([]byte("input1"))
+	acc2 = sdk.AccAddress([]byte("input2"))
+	acc3 = sdk.AccAddress([]byte("input3"))
+	acc4 = sdk.AccAddress([]byte("input4"))
+)
+
+// shared setup
+type TypesTestSuite struct {
+	suite.Suite
+
+	address []sdk.AccAddress
+	app     *simapp.SimApp
+	ctx     sdk.Context
+	params  types.AccountKeeper
+}
+
+func (suite *TypesTestSuite) SetupTest() {
+	suite.app = simapp.Setup(false)
+	suite.ctx = suite.app.BaseApp.NewContext(false, tmproto.Header{})
+	suite.params = suite.app.AccountKeeper
+
+	queryHelper := baseapp.NewQueryServerTestHelper(suite.ctx, suite.app.InterfaceRegistry())
+	types.RegisterMsgServer(queryHelper, &types.UnimplementedMsgServer{})
+
+	suite.address = []sdk.AccAddress{acc1, acc2, acc3, acc4}
+}
+
+func TestTypesTestSuite(t *testing.T) {
+	suite.Run(t, new(TypesTestSuite))
+}
+
+func (suite *TypesTestSuite) TestMsgSendRoute() {
```

Comment:
what's the purpose of this test?

Reply:
this test returns the name of the module and a human-readable string for the message